fixtures/github_authorization.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+@pytest.fixture
+def github_authorization(github_tok):
+    """Returns the authorization token for github
+    """
+    headers = {"Authorization": "token " + github_tok, "Accept": "application/vnd.github.v3.raw"}
```

Comment:
Do you want to separate the `Accept` header from the `Authorization` or perhaps rename the fixture since the former is separate from authorization??